model/labelset.go

```diff
@@ -150,6 +152,21 @@ func (ls LabelSet) FastFingerprint() Fingerprint {
 	return labelSetToFastFingerprint(ls)
 }
 
+func (l *LabelSet) MarshalEasyJSON(w *jwriter.Writer) {
```

Comment:
nit: Since maps are reference types and we're not mutating the labelset, is the pointer receiver useful here? A value receiver would allow just saying `range l` instead of `range *l` below.

Reply:
@juliusv looks like it won't matter, I was worried about copies etc. but it seems that the compiler is smart enough-- changed.